content/guides/getstarted/how-to-create-a-token.md

```diff
@@ -22,270 +22,261 @@ keywords:
 ```
 
-You will need to replace this with your own keypair in the following steps Then
-we configure the the Solana CLI to use to use the keypair we just created:
+<Callout type="note">
+Replace the `bos` public key above with your mint authority public key in the following steps!
+</Callout>
+
+Keep the contents of the mint authority keypair file safe. Whoever owns the key
+will be able to mint tokens, update metadata, and potentially freeze token
+accounts!
+
+Configure the Solana CLI to use to use the keypair we just created, for example:
 
 ```bash
-solana config set --keypair keyH23FC3gG4miLPCTTDWuD9PDX6E6V9kBk681ZnvQm.json
+solana config set --keypair bosy1VC2BH2gh5fdXA3oKn53EuATLwapLWC4VR2sGHJ.json
 ```
 
-Make sure to keep the contents of the keypair file safe. Whoever owns the key
-will be able to control the token mint and be able to mint tokens, update
-metadata and potentially freeze token accounts. From now on, all commands you
-run in the CLI will use this keypair by default. You can see the current
-configuration by running:
+We will also set the Solana CLI to use devnet:
 
 ```bash
-solana config get
+solana config set --url devnet
 ```
 
-Then we create a nice account address where our token mint will be saved:
+From now on, all transactions you run from the CLI will be signed using the
+`bos` keypair and be on devnet.
+
+You can see the current configuration by running:
 
 ```bash
-solana-keygen grind --starts-with nic:1
+solana config get
 ```
 
-In my case it created this keypair:
-`niceG6oxHmPcXVdNaUoECzWXn8Jz8fA5Q99QauJ4Gun.json`
+![`solana config get` showing devnet and our mint authority account](/assets/guides/make-a-token/solana-config-get.png)
 
-Make sure to replace this with your own keypair in the following steps.
+## Get some Devnet SOL for the mint authority
 
-We will also set the Solana CLI to use devnet for this example:
+![Solana Faucet](/assets/guides/make-a-token/solana-faucet.png)
+
+For Solana devnet we will just be able to get some free SOL,
+[the DevNet faucet](https://faucet.solana.com) If you get rate limited, follow
+this
+[guide to get devnet SOL](/content/guides/getstarted/solana-token-airdrop-and-faucets.md).
+Remember to fund your mint authority account - i.e. the keypair that starts with
+`bos` in your `new-token` folder. You can also just run:
 
 ```bash
-solana config set -ud
+solana address
 ```
 
-This will set the default RPC cluster to Solana devnet
-`https://api.devnet.solana.com`. From now on all commands will be executed on
-the devnet. To release your token on mainnet you will need to perform all step
-from here again but set the cluster to mainnet using `-um` flag instead of
-`-ud`.
+For mainnet you will need to fund the account with actual SOL. You can buy SOL
+with regular currency in places like centralized exchanges, crypto on-ramps, or
+swap other tokens for SOL on decentralized exchanges.
 
-## Fund the account
+## Make a Mint Address (`mnt`)
```

Comment:
```suggestion
## Make a Mint Address
```

The url gets parsed as object-object:
`/how-to-create-a-token#make-a-mint-address-object-object`

Reply:
Good spot!